src/profiles.js

```diff
@@ -39,6 +39,14 @@ const hexStringToBuffer = (hex) => {
 * and return a `Buffer` result
 */
 const encodes = {
+  /**
+  * @param {string} value
+  * @return {Buffer}
+  */
+  skynet: (value) => {
+    const multihash = multiH.encode(hexStringToBuffer(value), 'keccak-256');
+    return new CID(1, 'skynet-ns', multihash).buffer;
```

Comment:
This is still not working, even after updating all of the multicodec libraries. It's not working because obviously it depends on the `cids` package which we have not updated. Before going further I want to first make sure we even need a CID? I am not aware on what that is and why we would need it. 

More information here as well: https://github.com/multiformats/multicodec/pull/198

Reply:
`skynet-ns` which you added into multicodec looks like a protocode (or namespace), but CID should take content type which you didn't seem to add as [swarm team added](https://github.com/multiformats/multicodec/commit/35cfe5843232e062ffcd04d99238609c6783fb55)

Reply:
`Before going further I want to first make sure we even need a CID? I am not aware on what that is and why we would need it.` 

I think we do, otherwise we won't be able to differentiate whether which protocol the given data is and CID seems prepending these protocol names.

More detail is on https://github.com/ethereum/EIPs/blob/master/EIPS/eip-1577.md